cppSettings.gradle

```diff
@@ -260,5 +266,22 @@ subprojects {
                 linker.args niLibrary
             }
         }
+
+        // This task sets up the shared libraries to be stripped
+        ext.debugStripSetup = { project->
+            if (!project.hasProperty('debug')) {
+                project.tasks.whenObjectAdded { task ->
+                    if (task.name.contains('link') && task.name.contains('SharedLibrary')) {
```

Comment:
We've had names change case and break our stuff too often. Compare case insensitively.